xdem/terrain.py

```diff
@@ -59,7 +59,7 @@ def _get_quadric_coefficients(
     L = resolution
 
     # Allocate the output.
-    output = np.empty((12,) + dem.shape, dtype=dem.dtype) + np.nan
```

Comment:
Is there any reason why the dtype argument has been removed?

Reply:
Edited the answer in the PR description.